tests/console_test.go

```diff
@@ -41,57 +37,28 @@ var _ = Describe("Console", func() {
 
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
-	var vm *v1.VirtualMachine
-	var dial func(vm string, console string) *websocket.Conn
+	virtConfig, err := kubecli.GetKubevirtClientConfig()
+	tests.PanicOnError(err)
 
 	BeforeEach(func() {
 		tests.BeforeTestCleanup()
-
-		vm = tests.NewRandomVMWithSerialConsole()
-
-		dial = func(vm string, console string) *websocket.Conn {
-			wsUrl, err := url.Parse(flag.Lookup("master").Value.String())
-			Expect(err).ToNot(HaveOccurred())
-			wsUrl.Scheme = "ws"
-			wsUrl.Path = "/apis/kubevirt.io/v1alpha1/namespaces/" + tests.NamespaceTestDefault + "/virtualmachines/" + vm + "/console"
-			wsUrl.RawQuery = "console=" + console
-			c, _, err := websocket.DefaultDialer.Dial(wsUrl.String(), nil)
-			Expect(err).ToNot(HaveOccurred())
-			return c
-		}
 	})
 
 	Context("New VM with a serial console given", func() {
+		It("should be returned that we are running cirros", func() {
+			vm := tests.NewRandomVMWithSerialConsole()
 
-		It("should be allowed to connect to the console", func(done Done) {
```

Comment:
Notice I removed this functional test entirely. I don't see any value in keeping a test that just verifies console access works when all our other console based tests exercise this.

If console access is broken, we'll be able to tell that by the location of the failure in our other console tests. We can tell if it failed at connecting or failed sending/receiving.